main.go

```
@@ -0,0 +1,73 @@
+func executeHandler(w http.ResponseWriter, r *http.Request) {
+	// Command injection vulnerability
+	cmd := r.URL.Query().Get("cmd")
+
+	// Unsafe command execution
+	output, _ := exec.Command("bash", "-c", cmd).Output()
+
+	// Sensitive data exposure - returning command output directly
+	fmt.Fprintf(w, "Command output: %s", string(output))
+}
+
+func readSecretFile() string {
```

golangci-lint (1.62.2)

func `readSecretFile` is unused

(unused)